### PR TITLE
Mobile version project

### PR DESCRIPTION
**Pull request by Kwamay**
-[ ] Add image folders to the project
-[ ] Add files (linters) and other folders to the project
-[ ] Add LICENSE and README.md 
-[ ] Add html and css to create the mobile version
-[ ] Add gitignore file to project

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi @Kwamay, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️

   - You have submitted a project that has failing linters, Kindly make sure your linters are all passing before submitting the PR.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

File: index.html [hunk 1]
```diff
@@ -1,76 +1,290 @@
...
+          sign-ups required.
+        </p>
+        <ul class="language">
+          <li class="language-type">HTML</li>
+          <li class="language-type">CSS</li>
+          <li class="language-type">JavaScript</li>
+        </ul>
+        <button class="project-button">See project</button>
+      </div>
+    </section>
```
Comment (braincee) [inline index.html]: - [x] Please I highly suggest wrapping the whole work section with a `section` tag and CSS property `display: grid` per project requirements. I found out that you have wrapped each work card with `section` tags but you must instead wrap the work container with `section` tags 😄 

Example:

```
   <section class="second-phase">
      <div class="second-page">
        <img
          src="images/2nd-phase/pic_on_screen.png"
          alt="image"
          height="220px"
          width="295px"
        />

        <div class="bar-word">Tonic</div>
        <nav class="nav-list">
          <ul class="nav-items">
            <li class="canopy">CANOPY</li>

            <li class="back-end">BACK END DEV</li>
            <li class="year">2015</li>
          </ul>
        </nav>
        <p class="text">
          A daily selection of private personalized reads; no accounts or
          sign-ups required.
        </p>
        <ul class="language">
          <li class="language-type">HTML</li>
          <li class="language-type">CSS</li>
          <li class="language-type">JavaScript</li>
        </ul>
        <button class="project-button">See project</button>
      </div>
             <div class="second-page">
        <img
          src="images/2nd-phase/Avail.png"
          alt="image"
          height="220px"
          width="295px"
        />

        <div class="bar-word">Multi-Post Stories</div>
        <nav class="nav-list">
          <ul class="nav-items">
            <li class="canopy">CANOPY</li>

            <li class="back-end">BACK END DEV</li>
            <li class="year">2015</li>
          </ul>
        </nav>
        <p class="text">
          A daily selection of private personalized reads; no accounts or
          sign-ups required.
        </p>
        <ul class="language">
          <li class="language-type">HTML</li>
          <li class="language-type">CSS</li>
          <li class="language-type">JavaScript</li>
        </ul>
        <button class="project-button">See project</button>
      </div>
          <div class="second-page">
        <img
          src="images/2nd-phase/gym.png"
          alt="image"
          height="220px"
          width="295px"
        />

        <div class="bar-word">Tonic</div>
        <nav class="nav-list">
          <ul class="nav-items">
            <li class="canopy">CANOPY</li>

            <li class="back-end">BACK END DEV</li>
            <li class="year">2015</li>
          </ul>
        </nav>
        <p class="text">
          A daily selection of private personalized reads; no accounts or
          sign-ups required.
        </p>
        <ul class="language">
          <li class="language-type">HTML</li>
          <li class="language-type">CSS</li>
          <li class="language-type">JavaScript</li>
        </ul>
        <button class="project-button">See project</button>
      </div>
    </section>
```

File: index.html [hunk 2]
```diff
@@ -1,76 +1,290 @@
...
+    <section class="second-phase">
+      <div class="second-page">
```
Comment (braincee) [inline index.html]: - [x] Please I suggest wrapping the `second-page` class which represents each card of the work section with CSS Flexbox properties. This implementation should reflect on all the `second-page` classes on the `index.html`. 😄

File: index.html [hunk 3]
```diff
@@ -1,76 +1,290 @@
...
+        <img
+          src="images/2nd-phase/Avail.png"
+          alt="image"
+          height="220px"
+          width="295px"
+        />
+
+        <div class="bar-word">Multi-Post Stories</div>
```
Comment (braincee) [inline index.html]: - [x] Please I suggest replacing the `div` tag with `h2-h6` tags to properly implement HTML semantics for a better user experience. This is [link](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) for more information on Semantics. This implementation should apply to other project titles (eg. Tonic). 😄

File: style.css [hunk 4]
```diff
@@ -113,3 +136,190 @@ body {
...
+  border: 1px solid #396df2;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-size: 17px;
+  line-height: 24px;
+  font-weight: 500;
+  cursor: pointer;
+  padding: 12px;
+}
```
Comment (braincee) [inline style.css]: - [x] Please I highly suggest implementing button interactions( `hover`, `enabled`, `pressed`) on all buttons per project requirements. 😄 

**This is a screenshot of the Figma Design for button interactions**


Screenshot (321)

File: index.html [hunk 5]
```diff
@@ -1,76 +1,290 @@
...
+        <ul class="language">
+          <li class="language-type">HTML</li>
+          <li class="language-type">CSS</li>
+          <li class="language-type">JavaScript</li>
+        </ul>
+        <button class="project-button">See project</button>
+      </div>
+    </section>
+
+    <div class="about">
```
Comment (braincee) [inline index.html]: - [x] Please I highly suggest wrapping the whole `About Myself` section in a container with a `section` tag per the project requirements. 😄

File: index.html [hunk 6]
```diff
@@ -1,76 +1,290 @@
...
+      <h1 class="about-bigger-text">About Myself</h1>
```
Comment (braincee) [inline index.html]: - [x] Please I highly suggest replacing the `h1` tag of the `About Myself` text with `h2-h6` tags to implement HTML semantics for a better user experience. 😄

File: index.html [hunk 7]
```diff
@@ -1,76 +1,290 @@
...
+      <li class="icons-li">
+        <a href="#" class="icons-link">
+          <img src="images/icon_image/medium.png" alt="Hand" class="medium"
+        /></a>
+      </li>
+    </ul>
+
+    <button class="resume-button">Get my resume</button>
+
+    <ul class="about-list">
```
Comment (braincee) [inline index.html]: - [x] Please I highly suggest aligning the content of `about-list` class with CSS flexbox properties to implement a better user experience on the devices like tablets and desktops. 😄

File: index.html [hunk 8]
```diff
@@ -1,76 +1,290 @@
...
+        /></a>
+      </li>
+      <li class="icons-li">
+        <a href="#" class="icons-link">
+          <img src="images/icon_image/medium.png" alt="Hand" class="medium"
+        /></a>
+      </li>
+    </ul>
+
+    <button class="resume-button">Get my resume</button>
```
Comment (braincee) [inline index.html]: - [x] Please I highly suggest implementing button interactions(`hover`, `enabled`, `pressed`) on your `Get my resume` button per project requirements. 

**This is a screenshot of the Figma Design for button interactions**

Screenshot (321)

**Review by braincee — CHANGES_REQUESTED** [review]
Hello @Kwamay 👋 , I am @braincee your Code Reviewer

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

HIGHLIGHTS
------

 - GitHub flow rightfully used ✅
 - Meaningful title and summary for PR ✅
 - No linter errors ✅

REQUIRED CHANGES ♻️
------
**_Please check the comments under the review._**

OPTIONAL CHANGES
--------
**_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @braincee in your question so I can receive the notification.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

----

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 9]
```diff
@@ -113,3 +136,222 @@ body {
...
+
+.second-phase {
+  margin: 114px 24px 0;
+}
+
+.second-page {
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding: 16px 16px 28px;
+  display: flexbox;
```
Comment (topeogunleye) [inline style.css]: - [x] As suggested by the previous reviewer, I kindly suggest using CSS Flexbox properties on the second-page class which represents each card of the work section. The css property value of flexbox is wrong and nothing will be applied on the html as seen in the image below:


The correct way to apply the property would be:

```
.second-page {
    display: flex;
    flex-driection: column: /* To align the main axis vertically */ 
}
```

File: style.css [hunk 10]
```diff
@@ -113,3 +136,222 @@ body {
...
+
+.language-type {
+  background-color: #ebebff;
+  padding: 4px 12px;
+  border-radius: 8px;
+  margin-right: 8px;
+}
+
+.project-button {
+  color: #396df2;
```
Comment (topeogunleye) [inline style.css]: - [x] I kindly suggest that you add the correct button interaction for enable (default state) as shown in the Figma template below:

Screenshot from 2022-04-27 19-12-50

the button on the enabled state has the `color: #6070FF;` not `color: #396df2`

### EXAMPLE:
**for enable(default button) state**
```
button {
   color: #6070FF;
}
```

File: style.css [hunk 11]
```diff
@@ -113,3 +136,222 @@ body {
...
+  border-bottom: 1px solid #dfe1e6;
+}
+
+.language-words {
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+button:hover {
```
Comment (topeogunleye) [inline style.css:345]: - [x] I kindly suggest that you add the correct button interactions for hover as shown in the Figma template below:

Screenshot from 2022-04-27 19-12-50

the button on hover has the `background-color: #6070FF;` not `background-color: #396df2;`

### EXAMPLE:
```
**for hover state**
```
button:hover {
    background-color: #6070FF;
}
```

File: style.css [hunk 12]
```diff
@@ -113,3 +136,222 @@ body {
...
+  background-color: #396df2;
+  color: white;
+}
+
+button:active {
```
Comment (topeogunleye) [inline style.css:350]: - [x] I kindly suggest that you add the correct button interactions for active (when it's pressed) as shown in the Figma template below:

Screenshot from 2022-04-27 19-12-50

the button on hover has the  `background-color: #2230D2;` not `background-color: darkblue;`

### EXAMPLE:
```
**for pressed state**
```
button:active {
   background-color: #2230D2;
}
```

File: style.css [hunk 13]
```diff
@@ -113,3 +136,222 @@ body {
...
+.about-list {
+  display: flexbox;
+  list-style: none;
+  padding: 10px;
+  margin-left: 24px;
+  margin-right: 36px;
+}
+
+.resume-button {
+  color: #396df2;
```
Comment (topeogunleye) [inline style.css]: - [x] I kindly suggest that you add the correct button interaction for enable (default state) as shown in the Figma template below:

Screenshot from 2022-04-27 19-12-50

the button on the enabled state has the `color: #6070FF;` not `color: #396df2`

### EXAMPLE:
**for enable(default button) state**
```
button {
   color: #6070FF;
}
```

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @Kwamay, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* Wrapped the whole work section with a section tag ✔️ 
* Used display grid on work section as suggested ✔️ 
* Replaced div tag with h2-h6 tags to properly implement HTML semantics ✔️ 
* Replaced the h1 tag of the About Myself text with h2-h6 ✔️ 
* No linter errors. :heavy_check_mark:
* Followed Github flow :heavy_check_mark:

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
- For future pull requests, please notice that your branch name `mobile-br` does not follow the convention of being named after a feature or action. For example, you could name your branch added-mobile-br. This makes it easier to track the changes and the purpose of each branch.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 13 above)
Comment (Kwamay) [reply style.css]: @topeogunleye thanks for the review...

**Review by Cabraham1 — APPROVED** [review]
Hi @braincee  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**